docs/User-Guide_Fine-Tuning.md

```diff
@@ -1,30 +1,30 @@
 # How to customize keyboard, time zone?
 
```

Comment:
Perhaps this entire section can be simply changed with a link to armbian-config.md?

Reply:
I think some people like editing such things by hand. Though I added the hint for armbian-config